modules/core/perf/opencl/perf_arithm.cpp

```diff
@@ -1017,11 +1017,11 @@ OCL_PERF_TEST_P(ConvertScaleAbsFixture, ConvertScaleAbs,
 typedef Size_MatType PatchNaNsFixture;
 
 OCL_PERF_TEST_P(PatchNaNsFixture, PatchNaNs,
-                ::testing::Combine(OCL_TEST_SIZES, OCL_PERF_ENUM(CV_32FC1, CV_32FC4)))
+                ::testing::Combine(OCL_TEST_SIZES, OCL_PERF_ENUM(CV_32FC1, CV_32FC4, CV_64FC1, CV_64FC4)))
```

Comment:
C3 case is the most popular in cv and 3d ops. It makes sense to have it in test suite too.